cli/main.rs

```diff
@@ -1148,6 +1164,10 @@ async fn run_command(
   flags: Flags,
   run_flags: RunFlags,
 ) -> Result<i32, AnyError> {
+  if flags.check == CheckFlag::All && !flags.has_check_flag {
+    info!("{} In future releases deno run will not automatically type check without the --check flag.", colors::yellow("Warning"));
+  }
```

Comment:
This seems a bit extreme. Don't think we should do this warning.

Reply:
See https://github.com/denoland/deno/issues/14071